hugo/layouts/partials/home/hero.html

```diff
@@ -0,0 +1,27 @@
+<header class="home-hero">
+    <div class="home-hero__wrap">
+        <div class="home-hero__info">
+            <h1>{{ .title }}</h1>
+            <p>{{ .desc }}</p>
+            <div class="home-hero__info__links">
```

Comment:
Classname is not css matters. one is `btn-pill-white` and the other is `btn-pill-white-outline`